src/gtfs/read.rs

```diff
@@ -25,6 +25,10 @@ fn default_agency_id() -> String {
     "default_agency_id".to_string()
 }
 
+trait PrefixId {
```

Comment:
I don't get the idea of the trait.

Reply:
me neither :p, at first code was common between the 2 struct